transformer/kubernetes/apiresource/argocdapplication.go

```diff
@@ -70,6 +74,18 @@ func (*ArgoCDApplication) createNewResource(irApplication irtypes.Application, t
 		clusterServer = deployToSameCluster
 	}
 	appGVK := v1alpha1.ApplicationSchemaGroupVersionKind
+	prob := qatypes.Problem{}
+	prob.ID = common.JoinQASubKeys(parameterizer.ParamQuesIDPrefix, `"`+appGVK.GroupVersion().String()+`"`, `"`+appGVK.Kind+`"`, "destNamespace")
+	prob.Type = qatypes.InputSolutionFormType
+	prob.Desc = "Enter destination namespace for argo cd pipeline"
+	ques, err := qaengine.FetchAnswer(prob)
+	if err != nil {
+		logrus.Errorf("failed to ask a question to the user in order to parameterize a k8s resource: %+v\nError: %q", prob, err)
```

Comment:
we need to return on error.

Don't use the `FetchAnswer` function use the `FetchStringAnswer` function https://github.com/konveyor/move2kube/blob/817b8c901e9d3622a7fcbf6db55a34a08427ca90/qaengine/engine.go#L210

Example: https://github.com/konveyor/move2kube/blob/817b8c901e9d3622a7fcbf6db55a34a08427ca90/transformer/kubernetes/apiresource/triggertemplate.go#L91C8-L97